src/scripts/keyboardNavigation.js

```diff
@@ -178,6 +179,7 @@ export function enable() {
                 break;
 
             case 'Escape':
+                stopMultiSelect();
```

Comment:
IMO, this should be handled by the multiselect controller and prevent the default action so that the generic keyboard handler (this) [ignores the event](https://github.com/jellyfin/jellyfin-web/blob/0327f3317ef9a9c95f85c6d40fd0878cfd268995/src/scripts/keyboardNavigation.js#L128).